config/secrets.yml

```diff
@@ -1,5 +1,5 @@
 default: &default
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: 8370c4c3531fe72f6988d1a90fc8aa0be2f38492340ac7ec78cf7875797e19bd788e28a1e28ffebb261ba958c387298b082b708c45e46922c1268b16ea5eff15
```

Comment:
For security reasons it is not recommended to commit the secret key base. You can find more info [here](http://stackoverflow.com/questions/25426940/what-is-the-use-of-secret-key-base-in-rails-4)